ports/zephyr/ai.c

```diff
@@ -1,7 +1,6 @@
 /**************************************************************************
  *
  * Copyright (C) 2005 Steve Karg <skarg@users.sourceforge.net>
- * Copyright (C) 2011 Krzysztof Malorny <malornykrzysztof@gmail.com>
```

Comment:
For legal reasons copyright statements must not be removed from files.

Reply:
It seems that the changes in this file amount to only two things:

1. Use a memory allocator defined for the Zephyr system (which should come from a dedicated memory pool instead of using `k_malloc()` and be common across objects).
2. Change to use a BACnet-stack defined Keylist instead of an array - which should be a build option and generic across ports.

Is there anything else this file is supposed to do?